### PR TITLE
Add simpleCookie

### PR DESCRIPTION
**Pull request by tevko**
## Profile of the lib

* Git repository (required): https://github.com/tevko/simpleCookie
* Official website (optional, not the repository): N/A
* NPM package url (optional): https://www.npmjs.com/package/simplecookie
* License and its reference: [MIT](https://github.com/tevko/simpleCookie/blob/master/LICENSE)
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 2
  * Count of stars: 67
  * Count of forks: 1
* NPM download stats (optional): [https://npm-stat.com/](https://npm-stat.com/charts.html?package=simplecookie&from=2019-11-22&to=2019-12-22)
  * Downloads in the last day: 26
  * Downloads in the last week: 281
  * Downloads in the last month: 1321

## Essential checklist

* [x] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [ ] Has valid tags for each versions (for git auto-update)
* [ ] Auto-update setup
* [ ] Auto-update target/source is valid.
* [ ] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [ ] The parent of the commit(s) in the PR is not older than 3 days.
* [ ] Pull request is sent from a non-master branch with a meaningful name.
* [ ] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/simpleCookie/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "simplecookie",
+  "version": "1.0.1",
+  "description": "A small javascript utility to make handling cookies simple",
+  "main": "simpleCookie.min.js",
```
Comment (MattIPv4) [inline ajax/libs/simpleCookie/package.json]: This property should be `filename`

File: ajax/libs/simpleCookie/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "type": "git",
+    "url": "https://github.com/tevko/simpleCookie.git"
+  },
+  "keywords": [
+    "javascript",
+    "cookie"
+  ],
+  "author": "Tim Evko",
+  "license": "ISC",
+  "bugs": {
```
Comment (MattIPv4) [inline ajax/libs/simpleCookie/package.json]: This property is not needed

File: ajax/libs/simpleCookie/package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "url": "https://github.com/tevko/simpleCookie/issues"
+  },
+  "homepage": "https://github.com/tevko/simpleCookie",
+  "devDependencies": {
```
Comment (MattIPv4) [inline ajax/libs/simpleCookie/package.json]: This property is not needed

**Comment by tevko** [comment]
@MattIPv4 done!

File: ajax/libs/simpleCookie/package.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "simplecookie",
+  "version": "1.0.1",
```
Comment (MattIPv4) [inline ajax/libs/simpleCookie/package.json]: Please don't include this attr, the bot will populate this itself when it pulls in the version files.

File: ajax/libs/simpleCookie/package.json [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tevko/simpleCookie.git"
+  },
+  "keywords": [
+    "javascript",
+    "cookie"
+  ],
+  "author": "Tim Evko",
+  "license": "ISC",
```
Comment (MattIPv4) [inline ajax/libs/simpleCookie/package.json]:
```suggestion
  "license": "MIT",
```
License in the package appears to be MIT

File: ajax/libs/simpleCookie/package.json [hunk 6]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "simplecookie",
+  "version": "1.0.1",
+  "description": "A small javascript utility to make handling cookies simple",
+  "filename": "simpleCookie.min.js",
```
Comment (MattIPv4) [inline ajax/libs/simpleCookie/package.json]:
```suggestion
  "filename": "simpleCookie.js",
```
Only `simpleCookie.js` is in the package, no minified file.

**Comment by tevko** [comment]
@MattIPv4 additional changes implemented

File: ajax/libs/simpleCookie/package.json [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tevko/simpleCookie.git"
+  },
+  "keywords": [
+    "javascript",
+    "cookie"
+  ],
+  "author": "Tim Evko",
+  "license": "MIT",
```
Comment (MattIPv4) [inline ajax/libs/simpleCookie/package.json]: License is listed as ISC in the NPM package: https://www.npmjs.com/package/simplecookie

File: ajax/libs/simpleCookie/package.json [hunk 8]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "simplecookie",
```
Comment (MattIPv4) [inline ajax/libs/simpleCookie/package.json]:
```suggestion
  "name": "simpleCookie",
```
I think name should match the directory casing for consistency.

File: ajax/libs/simpleCookie/package.json (hunk 7 above)
Comment (xtuc) [inline ajax/libs/simpleCookie/package.json]:
```suggestion
  "license": "ISC",
```

**Pull request merged**